guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -12,7 +12,7 @@
     " - [Try the API live in a sandbox](#Try-it-live)\n",
```

Comment:
<ol><li>Trivial, should the arrows in <code>"Start &lt; Anaconda3 (64-bit) &lt; Anaconda Prompt"</code> point to the right?</li><li>Step 3 - hyperlink for <code>Map View</code> is wrong, it points to GIS object</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/490/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thanks @AtmaMani. Made those changes!